tensorflow/contrib/lite/testing/lstm_test.py

```diff
@@ -0,0 +1,274 @@
+import argparse
```

Comment:
Instead of adding a whole new file, could you add the test case into the existing generated examples test?

https://github.com/tensorflow/tensorflow/blob/e0d9dfd54b0f2f4a2a6637268aa176d1701ee8e9/tensorflow/lite/testing/generate_examples_lib.py#L3320

The test is runnable by 
```
bazel test //tensorflow/lite/testing:zip_test_lstm
```